AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -11,9 +11,13 @@ import UIKit
 
 class VideoModel: NSObject {
     private let maxRemoteVideoTileCount = 8
+    private let remoteVideoTileCountPerPage = 2
+
+    private var remoteVideoPageNumber = 1
```

Comment:
Not sure if this PageNumber is an index or count. Is this the index of current displaying page +1? If so, can we say `currentRemoveVideoPageIndex`?

Reply:
Yes this is `index of current displaying page +1`. I named it `remoteVideoPageNumber` on purpose because I think index always starts from 0, but this page number starts from 1.

But after a second thought, I think it may cause some confusion and it would be better to make it start from 0. Will rename the var.